### PR TITLE
Renamed caps to have YouiEngine defined in automationName rather than…

### PR DESCRIPTION
**Pull request by simongranger**
… platformName

Also cleaned up the project based on jshint

### CODE REVIEW

File: lib/commands/context.js [hunk 1]
```diff
@@ -10,7 +8,7 @@ const NATIVE_WIN = 'NATIVE_APP';
  * Actual MJSONWP command handlers
  * ------------------------------- */
 commands.getCurrentContext = async function () {
-  log.debug("getting context:" + NATIVE_WIN)
+  log.debug("getting context:" + NATIVE_WIN);
```
Comment (philgreenyouilabs) [inline lib/commands/context.js]: can we remove this?

File: lib/driver.js [hunk 2]
```diff
@@ -70,16 +70,16 @@ class YouiEngineDriver extends BaseDriver {
   }
 
   async createSession (caps) {
-    logger.debug("starting session")
+    logger.debug("starting session");
```
Comment (philgreenyouilabs) [inline lib/driver.js]: and this

File: lib/commands/find.js [hunk 3]
```diff
@@ -1,4 +1,3 @@
-import logger from '../logger';
 import { errors } from 'mobile-json-wire-protocol';
```
Comment (pfoster-youitv) [inline lib/commands/find.js]: shouldn't this be changed to the base driver?

Comment (simongranger) [inline lib/commands/find.js]: yes

File: lib/commands/context.js [hunk 4]
```diff
@@ -22,6 +19,7 @@ commands.getContexts = async function () {
 
 commands.setContext = async function (name) {
  // Always the same context
+ name = name; // to remove the jshint warning
```
Comment (pfoster-youitv) [inline lib/commands/context.js]: wat?

**Comment by pfoster-youitv** [comment]
`setContext` should do something else. Otherwise everything else looks ok.

**Comment by pfoster-youitv** [comment]
👍

**Pull request merged**